packages/quantic/cypress/integration/document-suggestion/document-suggestion-actions.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  DocumentSuggestionSelector,
+  DocumentSuggestionSelectors,
+} from './document-suggestion-selectors';
+
+function documentSuggestionActions(selector: DocumentSuggestionSelector) {
+  return {
+    clickSuggestion: (idx: number) =>
+      selector
+        .sectionTitle(idx)
+        .click()
+        .logAction(
+          'When clicking on the document suggestion at the specified index'
```

Comment:
Given you have the `idx` variable you could insert it in the logged text for that extra detail.